manifests/w/Webrandt/Ax/1.0.0/Webrandt.Ax.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Webrandt.Ax
+PackageVersion: 1.0.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Ax.exe
+  PortableCommandAlias: ax

```

```suggestion
  PortableCommandAlias: ax
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.DotNet.Runtime.8
```

**.NET Runtime** is required for running this application. It should be added as a package dependency.